test/integration/get_landing_page_data_test.go

```diff
@@ -95,13 +94,17 @@ func TestGetLandingPageData(t *testing.T) {
 
 		goldenFile := path.Join(goldenPath, c.goldenFile)
 		if generateGolden {
-			updateGolden(result, goldenFile)
+			updateGolden(result, goldenFile, true)
```

Comment:
nit: /* shard= */ true

(useful to know arg in the callsite)

Reply:
Done